kafka-zookeeper/docker-healthcheck

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+set -eo pipefail
+
+host="$(hostname --ip-address || echo '127.0.0.1')"
```

Comment:
filename and script convention per https://github.com/docker-library/healthcheck/blob/master/cassandra/docker-healthcheck